.github/workflows/release.yml

```diff
@@ -61,7 +61,7 @@ jobs:
           - arch: aarch64
             target-tupl: aarch64-unknown-linux-gnu
             dependencies: gcc-aarch64-linux-gnu protobuf-compiler libclang-dev g++-aarch64-linux-gnu
-            bindgenExtraClangArgs: "--sysroot=/usr/aarch64-linux-gnu -mfloat-abi=hard"
+            bindgenExtraClangArgs: "--sysroot=/usr/aarch64-linux-gnu"
```

Comment:
this is breaking all builds on ubuntu 24.04. Turns out `float-abi` isn't a valid param for `aarch64` targets